hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyCommitRequest.java

```diff
@@ -218,6 +224,75 @@ public void testValidateAndUpdateCache() throws Exception {
         omKeyInfo.getLatestVersionLocations().getLocationList());
   }
 
+  @Test
+  public void testAtomicRewrite() throws Exception {
+    if (getBucketLayout() == BucketLayout.FILE_SYSTEM_OPTIMIZED) {
+     // TODO - does not work with in FSO for now
+      return;
+    }
+
+    Table<String, OmKeyInfo> openKeyTable = omMetadataManager.getOpenKeyTable(BucketLayout.DEFAULT);
```

Comment:
This should actually use the method `getBucketLayout` as this test class is sub-classed for FSO and OBS buckets. As it was coded, it would have failed when we enabled FSO bucket testing but removing the temporary `if (getBucketLayout() == BucketLayout.FILE_SYSTEM_OPTIMIZED)` clause at line 229.